eng/pipelines/internal.yml

```diff
@@ -20,6 +20,11 @@ schedules:
     - master
   always: true
 
+# Remove this variable when fixed: https://github.com/dotnet/arcade/issues/3661
+variables:
+- name: _DotNetArtifactsCategory
+  value: .NETCore
```

Comment:
Shouldn't this be CORECLR?

I thought coreclr published to their own dotnet-coreclr blob feed:
https://github.com/dotnet/arcade/blob/0209480568cbbab2ebbf69e8e2c66e02dc5f1292/src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/PublishToPackageFeed.proj#L40

Reply:
Well... le sigh. Sounds like you are correct @safern. I think this was some miscommunication regarding what CoreCLR expected. Thanks for checking this out.